marin/post_training/README.md

```diff
@@ -0,0 +1,52 @@
+# Example RL Training Instructions
+
+This code is adopted from Charlie Snell's initial implementation of RL training in Jax on TPUs. The code uses the same worker for training and inference but uses continuous batching and parameter resharding between inference and training to optimize both steps.
+
+## General Setup
+
+This will change as we begin to use Marin's executor framework. For now, below are intructions for manually allocating a TPU node and launch experiments on that node.
```

Comment:
Isn't it easier to just do it using the executor framework, since you don't have to do allocate_tpu.sh, etc.?

Reply:
@dlwh and I will iterate on https://github.com/marin-community/marin/blob/rl/experiments/exp1403_rl_math.py for the integration after this PR is merged.